include/class.list.php

```diff
@@ -1458,6 +1458,9 @@ function delete() {
         if (!$this->isDeletable())
             return false;
 
+        // Disable Filters
+        Filter::disableFilters($this);
+
```

Comment:
Why not use signal here as well? Also you have to make sure parent actually deletes before disabling it.

```php
if (!$this->isDeletable() || !parent::delete())
    return false;

// Send deleted signal 

return true;
```

Reply:
I didn't call the signal here because we aren't currently auditing List Items (Including Ticket Statuses). We audit for lists when they are created, deleted, or the definition of the list is edited. I did make the change you noted for the code.

Reply:
Don't see why Audit becomes a factor here... signals don't exist because of audit plugin.

Reply:
It's just the way I was thinking about it, it made sense to use the signal for the times that we are auditing a delete as well as needing to disable filters because you could kill 2 birds with 1 stone, but since there's nothing to audit in this case, it seemed like it made sense to call disableFilters rather than to do it by using the signal.

Reply:
Please make the change...

Reply:
Updated